packages/docusaurus-plugin-content-docs/src/plugin-content-docs.d.ts

```diff
@@ -381,6 +385,8 @@ declare module '@docusaurus/plugin-content-docs' {
     draft?: boolean;
     /** Allows overriding the last updated author and/or date. */
     last_update?: FileChange;
+    /** Allows overriding the created author and/or date. */
+    create?: FileChange;
```

Comment:
I think that using "creation" would work well as it is a noun and fits well in existing function names e.g. `getFileCreation`. This would keep it grammatically consistent with last_update as well.